samples/bot-calling-meeting/csharp/Source/CallingBotSample/Services/TeamsRecordingService/ITeamsRecordingService.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading.Tasks;
+
+namespace CallingBotSample.Services.TeamsRecordingService
+{
+    public interface ITeamsRecordingService
```

Comment:
Please add doc comment to this interface explaining its purpose. Same for the members.